unstructured/documents/xml.py

```diff
@@ -78,13 +78,15 @@ def _read_xml(self, content):
             #     Please use  bytes input or XML fragments without declaration.
             except ValueError:
                 document_tree = etree.fromstring(content.encode(), self.parser)
-
             if "<pre>" and "</pre>" in content:
                 tree = etree.HTML(content)
                 for element in tree.xpath("//pre"):
                     if not element.text:
                         continue
-                    text_content = split_by_paragraph(element.text)
+                    text_content = [
```

Comment:
I think a regular for loop is more readable here, but the list comprehension is fine too if you want to keep it

Reply:
So, something like this?
```
for element in partition_text(text=element.text, paragraph_grouper=False):
    text_content.append(element.text)
```

Reply:
Yep! That'll do